tech-specs/src/explore/design/ibc.md

```diff
@@ -0,0 +1,18 @@
+# IBC integration
+
+IBC allows the ledgers to interact with each other. IBC is a protocol to agree the consensus state and to send/receive packets between ledgers.
+
+## Update to another ledger
+When a ledger executes a transaction including an update to another ledger, it sends a packet and executes the associated validation code like validity predicates to check if the packet has been received. The transaction writes the update to an internal storage subspace called IBC storage. IBC storage as an IBC module makes a packet and send it. After the transaction execution, not only validity predicates but also the IBC validation code associated with the update are executed. If the validation fails, e.g. the destination ledger fails to receive the update due to time out, the transaction is aborted.
+
+TODO The behaviour of the destination ledger
+
+## IBC storage
+IBC storage is an IBC module to make a packet and send it to the destination. A transaction stores an update for another ledger to the IBC storage instead of the storage into which other updates are written. The IBC storage doesn't persist the update into the DB, it makes a packet and send it to the destination ledger.
```

Comment:
It's not quite _storage_ updates for the other ledger, but rather messages which are handled in a particular way.